pkgs/development/python-modules/weasyprint/default.nix

```diff
@@ -44,12 +45,13 @@ buildPythonPackage rec {
       pango = "${pango.out}/lib/libpango-1.0${stdenv.hostPlatform.extensions.sharedLibrary}";
       pangocairo = "${pango.out}/lib/libpangocairo-1.0${stdenv.hostPlatform.extensions.sharedLibrary}";
       harfbuzz = "${harfbuzz.out}/lib/libharfbuzz${stdenv.hostPlatform.extensions.sharedLibrary}";
+      harfbuzz_subset = "${harfbuzz.out}/lib/libharfbuzz-subset${stdenv.hostPlatform.extensions.sharedLibrary}";
```

Comment:
For all of those we shoudl use lib.getLib

Reply:
I'd prefer not to block updates with unrelated improvements - as that aspect didn't change with this PR. You are very welcome to update them in a separate one :)